bin/lunamark

```diff
@@ -252,6 +252,48 @@ environment variable `LUNAMARK_EXTENSIONS` (see ENVIRONMENT below).
 :   Allow `#` instead of a digit for an ordered list enumerator
     (equivalent to `1`).
 
+`(-) task_list`
+:   GitHub-Flavored Markdown (GFM) task list extension to standard
+    bullet lists. Items starting with `[ ]`, `[x]` or `[X]` after the
+    bullet are processed as first-class structures by the Markdown reader
+    when this option is enabled, possibly allowing writers to handle
+    them more efficiently with a finer-grain logic.
+
+`(-) fancy_lists`
+:   Allow ordered list items to use a roman number or a letter
+    as anumerator, instead of a digit, and to use a closing parenthesis
+    as end delimiter, instead of a period.
+
+    Depending on the selected writer, the numbering scheme and delimeter
+    may be honored or just ignored.
+
+    If the `startnum` option is enabled, the starting value is converted
+    to decimal, when necessary, so that the list at least starts at the
+    appropriate value.
+
+    So in the minimal case, this option allows such lists to be processed,
+    albeit rendered as regular ordered lists.
+
+`(-) pandoc_extensions`
+:   Several Pandoc-like extensions are regrouped under that flag.
```

Comment:
I see several issues with having multiple extensions grouped into a single flag:

- The name `pandoc_extensions` is not very descriptive.
- There is no guarantee that the syntax extensions will not change in the future, which would forward-compatibility.
- It seems preferable to put the user through the hassle of specifying several options over giving him a single option that does not allow the user to opt out of syntax extensions that they don't want.

Please, split `pandoc_extensions` into the `strikeout`, `superscript`, `subscript`, `fenced_divs`, `bracketed_spans`, `raw_attribute`, `link_attributes` and `fenced_code_attributes` syntax extensions, [see also Pandoc's documentation][1]. Then, document the missing bits of the syntax extensions that are yet to be implemented, such as `#id`s in the `*_attributes` extensions as you mention in https://github.com/jgm/lunamark/pull/36#issuecomment-1187722927 and the lack of support for indirect images and direct/indirect links in the `link_attributes` extension, [see Pandoc's documentation][2].

 [1]: https://pandoc.org/MANUAL.html
 [2]: https://pandoc.org/MANUAL.html#extension-link_attributes

Reply:
Done.
- `#id` added (see below)
- Options split as distinct flags as requested + missing bits documented
- On the way, I noticed Pandoc allowed escaped spaces in super/subscripts (for some good reasons), so I added that rule too.